clusters/app.ci/supplemental-ci-images/README.md

```diff
@@ -28,3 +28,4 @@ These resources provide some supplemental images to be used in CI builds.
 - `ubi8-nginx`: Used by [crane-ui-plugin](../../../ci-operator/config/konveyor/crane-ui-plugin/)
 - `ubuntu.yaml`: Used by [kata-containers](../../../ci-operator/config/kata-containers)
 - `validation-images`: Used by [openshift/release](../../../ci-operator/config/openshift/release)
+- `hypershift`: Used by Hypershift-related workflows (e.g., ../../../ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml)
```

Comment:
Is it a link in MD like others?

Are we trying to modify the workflow?
Are the owners aware of the change?
https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/OWNERS

Reply:
Yes, we are modifying the workflow. It will be in another PR and also I will discuss with the owners.

Reply:
I duplicated the workflow to a new location which no longer requires the original owner's approval. Also, the document will come with the PR with workflows (https://github.com/openshift/release/pull/33635)